src/auth/user.service.ts

```
@@ -0,0 +1,53 @@
+    return this.prismaService.user.update({
+      where: { id: userId },
+      data,
+    });
+  }
+
+  async deleteUser(userId: number): Promise<User> {
+    return this.prismaService.user.delete({
+      where: { id: userId },
+    });
+  }
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)